src/tests_common.h

```diff
@@ -0,0 +1,42 @@
+/***********************************************************************
+ * Distributed under the MIT software license, see the accompanying    *
+ * file COPYING or https://www.opensource.org/licenses/mit-license.php.*
+ ***********************************************************************/
+
+#ifndef SECP256K1_TEST_COMMON_H
```

Comment:
91eac9c9411bf27607ffc1e114fec039a7fca00a: a typo:
```suggestion
#ifndef SECP256K1_TESTS_COMMON_H
```

Reply:
Done, thanks.